Futura/Futura/Either.swift

```diff
@@ -0,0 +1,24 @@
+/* Copyright 2018 Miquido
+
+ Licensed under the Apache License, Version 2.0 (the "License");
+ you may not use this file except in compliance with the License.
+ You may obtain a copy of the License at
+
+ http://www.apache.org/licenses/LICENSE-2.0
+
+ Unless required by applicable law or agreed to in writing, software
+ distributed under the License is distributed on an "AS IS" BASIS,
+ WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ See the License for the specific language governing permissions and
+ limitations under the License. */
+
+internal enum Result<T> {
+    case success(T)
+    case error(Error)
+}
+
+// TODO: remove Result in favor of Either
```

Comment:
Do you have marking TODOs as warnings? You could add a build phase for that, this could get forgotten otherwise.

`TAGS="TODO:|FIXME:"
echo "searching ${SRCROOT} for ${TAGS}"
cd "${SRCROOT}"
find "." -path "./Pods" -prune -o \( -name "*.swift" \) -print0 \
| xargs -0 egrep --with-filename --line-number --only-matching "($TAGS).*\$" \
| perl -p -e "s/($TAGS)/ warning: \$1/"`

or use a `#warning`

Reply:
I will change those to `#warning` I've completely forgot about them 😅